### PR TITLE
MINOR: Make 'Topic-Level Configs' a doc section for easier access

### PR DESCRIPTION
**Pull request by vahidhashemian**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5625/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5639/
Test PASSed (JDK 7 and Scala 2.11).

File: docs/configuration.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
   <p>More details about broker configuration can be found in the scala class <code>kafka.server.KafkaConfig</code>.</p>
 
-  <a id="topic-config" href="#topic-config">Topic-level configuration</a>
+  <h3><a id="topic-config" href="#topic-config">3.2 Topic-Level Configs</a></h3>
```
Comment (ijuma) [inline docs/configuration.html]: We should change the reference to be consistent with the other ones (i.e. no hyphen and plural).

**Review by ijuma — COMMENTED** [review]
Thanks for the PR, just one comment.

**Comment by vahidhashemian** [comment]
@ijuma Thanks for the feedback. The reference was updated in the new commit.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6037/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6052/
Test PASSed (JDK 7 and Scala 2.11).

**Review by ijuma — APPROVED** [review]
Thanks, LGTM. Merging to trunk and 0.11.0.

**Pull request closed without merging**